docs/docs/behaviors/mouse-emulation.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Mouse Emulation Behaviors
+sidebar_label: Mouse Emulation
+---
+
+## Summary
+
+Mouse emulation behaviors send mouse events. Currently, only mouse button presses are supported, but movement
+and scroll action support is planned for the future.
+
```

Comment:
```suggestion

:::caution
This feature will be automatically enabled when you use any of the following behaviors in your keymap and might require re-pairing your keyboard to your devices to work over bluetooth. If mouse behaviors are still not recognized by your device after doing that, you can try [these troubleshooting steps](../features/bluetooth.md#windows-connected-but-not-working).
:::
```
I think this is something that folks are likely to hit, to refresh the HID descriptors (should we mention that explicitly?)

Reply:
I wonder if we might want to call that out explicitly near the top of this doc page?

Reply:
You mean the HID descriptor change? Might be a good idea, though I am not confident enough on the details myself. Maybe worth expanding my caution note.

Reply:
@caksoylar Pushed some tweaks. Thoughts?

Reply:
LGTM, thanks.